Sources/Log.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+#include <QtDebug>
+#include <QFile>
+#include <QTextStream>
+#include <QUuid>
+#include "KVAbort.h"
+#include <QDateTime> 
```

Comment:
Headers which are used only in `Log.cpp` should be included in `Log.h` - instead they should be included in `Log.cpp`